src/Base64.test.tsx

```diff
@@ -0,0 +1,16 @@
+import Base64 from "./Base64";
+import {shallow} from 'enzyme';
+
+describe('test Base64 component', () => {
+    test('expected elements are rendered', () => {
+        const wrapper = shallow(<Base64 />);
+        expect(wrapper.find('#text-input')).toExist();
+        expect(wrapper.find('#encoding-select')).toExist();
+        expect(wrapper.find('#encoding-select')).toHaveValue('UTF-8');
```

Comment:
This fails due to the latest commit of main, which makes "_UTF-8" preselected.

Reply:
Done